fluid/mnist/model.py

```diff
@@ -181,15 +181,10 @@ def run_benchmark(model, args):
             "pass=%d, train_avg_acc=%f,train_avg_loss=%f, test_avg_acc=%f, elapse=%f"
             % (pass_id, train_avg_acc, train_avg_loss, test_avg_acc,
                (pass_end - pass_start)))
-
-        with open("train_acc_factor.txt", 'a+') as f:
-            f.write("%s\n" % train_avg_acc)
-        with open("train_cost_factor.txt", 'a+') as f:
-            f.write('%s\n' % [train_avg_loss])
-        with open("test_acc_factor.txt", 'a+') as f:
-            f.write("%s\n" % test_avg_acc)
-        with open("train_duration_factor.txt", 'a+') as f:
-            f.write('%s\n' % [pass_end - pass_start])
+        print ("kpis	train_acc	%f" % train_avg_acc)
```

Comment:
add a comment about this usage(specially for CE)

Reply:
ok